ui/page/rewardsVerify/index.jsx

```diff
@@ -4,20 +4,8 @@ import UserVerify from 'component/userVerify';
 import Page from 'component/page';
 import { useHistory } from 'react-router-dom';
 
-type Props = {
-  user: ?User,
-};
-
-function RewardsVerifyPage(props: Props) {
-  const { user } = props;
+function RewardsVerifyPage() {
   const { goBack } = useHistory();
-  const rewardsApproved = user && user.is_reward_approved;
-
-  React.useEffect(() => {
-    if (rewardsApproved) {
-      goBack();
-    }
-  }, [rewardsApproved]);
```

Comment:
I always find the automatic `goBack()` in `/$/rewardsVerify` when already verified quite confusing.  This seems like a good direction.

:small_blue_diamond: A few things to consider/check:
- If I'm verified and I enter `/$/rewardsVerify` (say, from clicking an old notification, or due to some sequence), the page still allows you to perform the verification actions.  
    - Minor, though.  And not necessarily needed in this PR, but can gray out and say "already verified" in the future.
- I vaguely recall there is reliance on the old behavior in one of the flows ... can't recall if it's sign-up, sign-in, comment-when-no-credits, or something else.  (Can ignore this for now, I'll try to dig).